src/components/10-atoms/toggle-switch/ui.test.js

```diff
@@ -81,27 +94,25 @@ test('should set correct background-color to inactive disabled toggle-switch-sli
       '.a-toggle-switch__slider'
     );
 
-    return new Promise(resolve => {
-      toggleSwitchSlider.addEventListener('transitionend', () =>
-        resolve(
-          window
-            .getComputedStyle(toggleSwitchSlider)
-            .getPropertyValue('background-color')
-        )
-      );
-    });
+    return window
+      .getComputedStyle(toggleSwitchSlider)
+      .getPropertyValue('background-color');
   }).with({ boundTestRun: t });
 
   await setProperties();
-  await t.expect(await getBackgroundColor()).eql('rgb(245, 245, 245)');
+
+  // Timeout is needed to wait until the animation has finished.
+  setTimeout(async () => {
+    await t.expect(await getBackgroundColor()).eql('rgb(245, 245, 245)');
+  }, 410);
 });
 
 test('should set correct background-color to active disabled toggle-switch-slider', async t => {
   const setProperties = ClientFunction(() => {
     const toggleSwitch = document.querySelector('axa-toggle-switch');
 
     toggleSwitch.active = true;
-    toggleSwitch.disabled = true;
+    toggleSwitch.disabled = false;
```

Comment:
old: true
new: false

--> is that correct? maybe raphi should review this PR.